benefit/data/sheets/CaseType/CaseType.json

```diff
@@ -1 +1,2 @@
 [{"LiveFrom": "01/01/2018", "ID": "Benefit", "Name": "SSCS Case v5.14.27_${CCD_DEF_E}", "Description": "SSCS Benefit Cases", "JurisdictionID": "SSCS", "PrintableDocumentsUrl": "http://ccd-data-store-api-prod.service.core-compute-prod.internal/callback/jurisdictions/SSCS/case-types/Benefit/documents", "RetriesTimeoutURLPrintEvent": "5,10,15", "SecurityClassification": "PUBLIC"}]
```

Comment:
Don't forget to bump the version number. Same in VERSION.yaml

Reply:
Version was bumped in both files, they are in sync